content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -15,6 +15,7 @@ card:
 
 This page contains a list of commonly used `kubectl` commands and flags.
 
+These instructions are for Kubernetes {{< skew currentVersion >}}. To check the version, use the `kubectl version` command.
```

Comment:
Nit: It would be a nice addition to include the 'v' prefix to the shortcode i.e.`v{{< skew currentVersion >}}`

Also, I feel enclosing this comment within {{< note >}} {{< /note >}} would enhance its visibility and importance.

Reply:
Done. Thank you for the suggestions @dipesh-rawat